### ISSUE TITLE
Exception when rendering big JSON template

### ISSUE THREAD

**Issue by betalb** [maghoff/mustachio#19]
I have large JSON file that has few template placeholders.

And it fails to render with below exception
```
RangeError: Maximum call stack size exceeded
    at Template.<anonymous> (C:\work\repos\pleeco-fluent-mock-cp\node_modules\mustachio\lib\template.js:24:12)
    at Generator.next (<anonymous>)
    at GeneratorStream._read (C:\work\repos\pleeco-fluent-mock-cp\node_modules\mustachio\lib\generator-stream.js:42:25)
    at chunk.value.then.result (C:\work\repos\pleeco-fluent-mock-cp\node_modules\mustachio\lib\generator-stream.js:64:10)
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

I've assembled simple app to reproduce this. If I remove 2 lines from template, exception goes away.

Template [here](https://github.com/betalb/attachments/blob/a7b072e2689a9d6bc458fc4df65c5b05f2ce17b3/mustachio.json)
index.js
```
const http = require('http');
const path = require('path');
const mustachio = require('mustachio');

const resolver = mustachio.resolver({
    root: path.join(__dirname, "assets"),
    suffixes: [ ".json" ]
});

const server = http.createServer((req, res) => {
  res.writeHead(200, {'Content-Type': 'application/json'});
  const templateStream = resolver('static-data', {}).stream();

  templateStream.pipe(res);
});

server.listen(3100);
```

Node version: v7.4.0
OS: Windows 10
npm version: 4.0.5
mustachio version: 1.0.0

**Comment by maghoff** [comment]
Hi, thanks for your detailed report. I have been able to reproduce this problem.

The problem seems to go away if I insert a few tags in the template. Do you see the same thing?

The limitation seems to be that the JS engine does not want too many string concatenations in one expression, so I'll try working around this in the library as well.

**Comment by maghoff** [comment]
I have committed a proposed fix to a branch now. Are you able to test if https://github.com/maghoff/mustachio/tree/issue-19 fixes your problem?

**Comment by maghoff** [comment]
I have now published version 1.0.1, which should fix this problem. I would like to hear back from you, @betalb if this fixes the problem for you.
